blocks/columns/columns.css

```diff
@@ -89,6 +81,14 @@
     padding: var(--spacing-medium);
   }
 
+  .columns.column-separater > div > div {
+    border-right: 1px solid var(--c-copy);
+  }
+
+  .columns.column-separater > div > div:last-child {
```

Comment:
Instead of this, we can handle it like this .columns.column-separater > div > div:not(:last-child){}
Also typo error "separator"